src/clj/harja/kyselyt/yllapitokohteet.sql

```diff
@@ -208,8 +160,6 @@ SELECT
   sijainti
 FROM yllapitokohdeosa ypko
   JOIN yllapitokohde ypk ON ypko.yllapitokohde = ypk.id
-                            AND urakka = :urakka
-                            AND sopimus = :sopimus
```

Comment:
Otin nämä rajaukset pois, sillä tuli vastaan tilanne, missä näitä ei anneta. Tarkistin, että palveluissa on käytetty vaadi-yllapitokohde-kuuluu-urakkaan aina kun se on tarpeellista.